packages/wonder-blocks-link/src/components/link.tsx

```diff
@@ -123,6 +123,16 @@ type CommonProps = AriaProps & {
      * the default `externalIcon`.
      */
     endIcon?: React.ReactElement<React.ComponentProps<typeof PhosphorIcon>>;
+
+    /**
+     * An optional prop that enables a
+     * [https://developer.mozilla.org/en-US/docs/Web/API/View_Transition_API](View
+     * Transition) for this navigation by wrapping the final state update in
+     * `document.startViewTransition()`.
+     *
+     * @see https://reactrouter.com/6.30.0/components/link#viewtransition
+     */
+    viewTransition?: boolean;
```

Comment:
How do we see the `prefers-reduced-motion` app setting fitting in with the `viewTransition`?  Would this prop be set based on that setting (similar to the `animated` prop)?

Reply:
Good question! this should be probably handled at the consumer level (probably using the `prefers-reduced-motion` mq when defining the view transition animations).

It's kind of annoying to do it this way, but it is a limitation of the VT API as they only allow applying styles globally.

Reply:
Would it be helpful to add documentation around the preferred way of handling it? (We can add it later on too when we learn more from integrating it into places 😄)

Reply:
That was my question too! We'll want to have a solid pattern for reducing motion.